docs/README.md

```diff
@@ -0,0 +1,325 @@
+# Design and implementation of Edge Virtualization Engine
+
+1. [Introduction](#introduction)
+2. [Edge Containers](#edge-containers)
+3. [Security](#security)
+4. [EVE Controller](#eve-controller)
+5. [Runtime Configuration](#runtime-configuration)
+6. [Installing EVE on Edge Nodes](#installing-eve-on-edge-nodes)
+7. [Runtime Lifecycle](#runtime-lifecycle)
+8. [Building EVE](#building-eve)
+9. [EVE Internals](#eve-internals)
+
+## Introduction
+
+Edge Virtualization Engine (EVE) is a secure by design operating system
+that supports running *Edge Containers* on compute devices deployed in
+the field. These devices can be IoT gateways, Industrial PCs or general
+purpose ruggedized computers and we will refer to them as *Edge Nodes*.
+EVE is based on a type-1 hypervisor and it does not directly support
+traditional POSIX-like application and processes. All applications
+running on EVE have to be represented as [Edge Containers](ECOS.md) which
+can be either Virtual Machines, Containers or Unikernels. EVE presents a
```

Comment:
"either / or" by definition means 2 options. We have 3. So we have to make it, "can be one of: Virtual Machines, Containers or Unikernels."

Reply:
fixed